src/Core/Utility/Path.re

```diff
@@ -11,8 +11,8 @@ let hasTrailingSeparator = path => {
   };
 };
 
-let toRelative = (~base, path) => {
```

Comment:
This is code added just for testing, with a default that will not be tested. Wouldn't it be better make the tests platform-specific so we test all the actual code paths on all relevant platforms instead?

Reply:
That's a reasonable concern - made the suggested change in the latest comment